CHANGELOG.rst

```diff
@@ -5,6 +5,8 @@ Changelog
 -------------------
 
 - Fix the slowness when retrieving a device via private vehicle API
+- Fix with_latest_events query that is taking too much time, used in Agency API
+- Fix bug of having dn_battery_pct in Telemetry field in eventRecord table
```

Comment:
I think both are obsolete now.

Reply:
The last one will be moved to a separate PR, but why the the first one about latest_events ? The query is still used in agency_api

Reply:
The performance issue still remains? My bad.